src/components/molecules/UserSearchBar/UserSearchBar.scss

```diff
@@ -70,9 +70,8 @@
 .user-search-results {
   position: absolute;
   width: 100%;
-  position: fixed;
   z-index: 380;
-  top: 60px;
+  top: 105%;
```

Comment:
[thinking, not really a review comment] Just thinking about responsive etc. We are at a point where a standard set of recommendations will be very helpful for us to do responsive layouts really well.

A place that answers questions like: what are the recommended places to use % and when to use pixels, how that affects responsive layouts, etc. etc. would be really handy for the team.

Reply:
In this case, its not due to responsiveness. 105% is telling the child (absolute) component to go 5% under the bottom edge of the parent (relative) component.

Same result would be with `bottom: -5%;`

If pixels were used for this result, you would need to know how tall the parent is and add a few more pixels to push the child out of the box of the parent